travis/tests/static/model/configuration/static.fac

```diff
@@ -0,0 +1,187 @@
+K8I = 3.0D-11 ;
+KR8 = K80/K8I ;
+FC8 = 0.41 ;
+NC8 = 0.75-1.27*(LOG10(FC8)) ;
+F8 = 10@(LOG10(FC8)/(1+(LOG10(KR8)/NC8)**2)) ;
+KMT08 = (K80*K8I)*F8/(K80+K8I) ;
+K90 = 1.4D-31*M*(TEMP/300)@-3.1 ;
+K9I = 4.0D-12 ;
+KR9 = K90/K9I ;
+FC9 = 0.4 ;
+NC9 = 0.75-1.27*(LOG10(FC9)) ;
+F9 = 10@(LOG10(FC9)/(1+(LOG10(KR9)/NC9)**2)) ;
+KMT09 = (K90*K9I)*F9/(K90+K9I) ;
+K100 = 4.10D-05*M*EXP(-10650/TEMP) ;
+K10I = 6.0D+15*EXP(-11170/TEMP) ;
+KR10 = K100/K10I ;
+FC10 = 0.4 ;
+NC10 = 0.75-1.27*(LOG10(FC10)) ;
+F10 = 10@(LOG10(FC10)/(1+(LOG10(KR10)/NC10)**2)) ;
+KMT10 = (K100*K10I)*F10/(K100+K10I) ;
+K1 = 2.40D-14*EXP(460/TEMP) ;
+K3 = 6.50D-34*EXP(1335/TEMP) ;
+K4 = 2.70D-17*EXP(2199/TEMP) ;
+K2 = (K3*M)/(1+(K3*M/K4)) ;
+KMT11 = K1 + K2 ;
+K120 = 2.5D-31*M*(TEMP/300)@-2.6 ;
+K12I = 2.0D-12 ;
+KR12 = K120/K12I ;
+FC12 = 0.53 ;
+NC12 = 0.75-1.27*(LOG10(FC12)) ;
+F12 = 10@(LOG10(FC12)/(1.0+(LOG10(KR12)/NC12)**2)) ;
+KMT12 = (K120*K12I*F12)/(K120+K12I) ;
+K130 = 2.5D-30*M*(TEMP/300)@-5.5 ;
+K13I = 1.8D-11 ;
+KR13 = K130/K13I ;
+FC13 = 0.36 ;
+NC13 = 0.75-1.27*(LOG10(FC13)) ;
+F13 = 10@(LOG10(FC13)/(1+(LOG10(KR13)/NC13)**2)) ;
+KMT13 = (K130*K13I)*F13/(K130+K13I) ;
+K140 = 9.0D-5*EXP(-9690/TEMP)*M ;
+K14I = 1.1D+16*EXP(-10560/TEMP) ;
+KR14 = K140/K14I ;
+FC14 = 0.36 ;
+NC14 = 0.75-1.27*(LOG10(FC14)) ;
+F14 = 10@(LOG10(FC14)/(1+(LOG10(KR14)/NC14)**2)) ;
+KMT14 = (K140*K14I)*F14/(K140+K14I) ;
+K150 = 8.6D-29*M*(TEMP/300)@-3.1 ;
+K15I = 9.0D-12*(TEMP/300)@-0.85 ;
+KR15 = K150/K15I ;
+FC15 = 0.48 ;
+NC15 = 0.75-1.27*(LOG10(FC15)) ;
+F15 = 10@(LOG10(FC15)/(1+(LOG10(KR15)/NC15)**2)) ;
+KMT15 = (K150*K15I)*F15/(K150+K15I) ;
+K160 = 8D-27*M*(TEMP/300)@-3.5 ;
+K16I = 3.0D-11*(TEMP/300)@-1 ;
+KR16 = K160/K16I ;
+FC16 = 0.5 ;
+NC16 = 0.75-1.27*(LOG10(FC16)) ;
+F16 = 10@(LOG10(FC16)/(1+(LOG10(KR16)/NC16)**2)) ;
+KMT16 = (K160*K16I)*F16/(K160+K16I) ;
+K170 = 5.0D-30*M*(TEMP/300)@-1.5 ;
+K17I = 1.0D-12 ;
+KR17 = K170/K17I ;
+FC17 = 0.17*EXP(-51/TEMP)+EXP(-TEMP/204) ;
+NC17 = 0.75-1.27*(LOG10(FC17)) ;
+F17 = 10@(LOG10(FC17)/(1.0+(LOG10(KR17)/NC17)**2)) ;
+KMT17 = (K170*K17I*F17)/(K170+K17I) ;
+KMT18 = 9.5D-39*O2*EXP(5270/TEMP)/(1+7.5D-29*O2*EXP(5610/TEMP)) ;
+KPPN0 = 1.7D-03*EXP(-11280/TEMP)*M ;
+KPPNI = 8.3D+16*EXP(-13940/TEMP) ;
+KRPPN = KPPN0/KPPNI ;
+FCPPN = 0.36 ;
+NCPPN = 0.75-1.27*(LOG10(FCPPN)) ;
+FPPN = 10@(LOG10(FCPPN)/(1+(LOG10(KRPPN)/NCPPN)**2)) ;
+KBPPN = (KPPN0*KPPNI)*FCPPN/(KPPN0+KPPNI) ;
+****************************************************** ;
+*;
+* Peroxy radicals. ;
+*;
+* WARNING: The following species do not have SMILES strings in the database. ;
+*          If any of these are peroxy radicals the RO2 sum will be wrong!!! ;
+****************************************************** ;
+*  ;
+RO2 = HOCH2CH2O2 + BUT2OLO2 + CH3O2 + CH3CO3 + HCOCH2O2 + MEKAO2 +
+      MEKBO2 + MEKCO2 + C2H5O2 + HOCH2CO3 + BIACETO2 + HO1CO3C4O2 +
+      CO2C3CO3 + CH3COCH2O2 + C2H5CO3 + HCOCO3 + HOC2H4CO3 ;
+*;
+* Reaction definitions. ;
+*;
+% (TEMP/300) : O = O3 ;
```

Comment:
Just as a note you don't need to have temperature dependence (unless that is part of what you want to test, of course). `k` can be simply `0.9` in this example.

Reply:
I left this in as it is doing no harm.